src/groovy/de/hbznrw/ygor/validators/Validator.groovy

```diff
@@ -77,11 +77,22 @@ class Validator {
    * @param abbreviation
    * @return
    */
-  static isValidAbbreviation(String abbreviation) {
+  static isValidAbbreviation(String abbreviation, Integer minLength = null, Integer maxLength = null) {
     if (StringUtils.isEmpty(abbreviation)) {
       return Status.MISSING
     }
-    if (abbreviation.length() > 1) {
+    if (maxLength == null){
+      // check for default maximum length of 1
```

Comment:
default max = 1 seems a bit short

Reply:
Exactly. This is why there is a field-specific maxLength now.
Problem is: some abbreviations are only allowed to have a size of 1 character.